src/calcium/CalciumTypes.jl

```diff
@@ -59,11 +59,36 @@ end
 #
 ################################################################################
 
+qqbar_field_doc = raw"""
+    QQBarField <: Field
+    QQBarFieldElem <: FieldElem
+
+The algebraic closure $\overline{\mathbb Q}$ of the rationals $\mathbb Q$ and
+its elements. This is a singleton type, meaning that there is a unique global
+instance of this field, which can be obtained via `algebraic_closure(QQ)`.
+
+```jldoctest
+julia> K = algebraic_closure(QQ)
+Field of algebraic numbers
+
+julia> sqrt(K(2))
+Root 1.41421 of x^2 - 2
+
+julia> QQBar(2//3)
```

Comment:
this still mentiones the singleton QQBar in this example (and three rows below). x-ref https://github.com/Nemocas/Nemo.jl/pull/1715